config.go

```diff
@@ -0,0 +1,168 @@
+package gex
+
+import (
+	"bytes"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/izumin5210/gex/pkg/manager"
+	"github.com/izumin5210/gex/pkg/manager/dep"
+	"github.com/izumin5210/gex/pkg/manager/mod"
+	"github.com/izumin5210/gex/pkg/tool"
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+)
+
+// Config specifies the configration for managing development tools.
+type Config struct {
+	OutWriter io.Writer
+	ErrWriter io.Writer
+	InReader  io.Reader
+
+	FS afero.Fs
+
+	WorkingDir   string
+	ManifestName string
+	BinDirName   string
+
+	Verbose bool
+	Logger  *log.Logger
+
+	mode Mode
+}
+
+// Defualt contains default configuration.
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported var Default should be of the form "Default ..."

Reply:
🙄